content/200-concepts/100-components/02-prisma-client/037-relation-queries.mdx

```diff
@@ -1079,7 +1006,7 @@ const update = await prisma.post.update({
 
 You can nest `create` or `createMany` inside an `update` to add new related records to an existing record. The following query adds two posts to a user with an `id` of 9:
 
-```ts
+```ts highlight=6-13;normal
```

Comment:
this seems glitchy:

<img width="536" alt="image" src="https://github.com/prisma/docs/assets/4058327/1653358b-5a9f-4d7d-9a7b-0a831a2c558d">